src/mapml/layers/TemplatedImageLayer.js

```diff
@@ -132,6 +139,7 @@ export var TemplatedImageLayer =  L.Layer.extend({
     onRemove: function (map) {
       this._clearLayer();
       map._removeZoomLimit(this);
+      this._layer.isUnChecked = true;
```

Comment:
why is `this._layer` added and what's is use? can't see where this is being used

Reply:
Yes it is from the previous commit I made, I simply forgot to delete this. What I was trying to do in my previous commit was to ensure the redraw() function should only be called inside the onAdd function when the layer is **re**-added to the map. I added it as an attribute of this._layer is simply because I found the templated image layer itself as well as its container will be cleared once I turn the layer off, so if I make it as an attribute of this, it can no longer be used when I add the layer back. Overall, thank you for bringing this up, I will remove it from the codes!